plugins/remote-config/api/api.js

```diff
@@ -370,7 +415,7 @@ plugins.setConfigs("remote-config", {
         parameter.conditions = parameter.conditions || [];
         processParamValue(parameter);
 
-        var maximumParametersAllowed = plugins.getConfig("remote-config").maximum_allowed_parameters;
+        var maximumParametersAllowed = params.qstring.isTest ? 4 : plugins.getConfig("remote-config").maximum_allowed_parameters;
```

Comment:
It's not a good idea to add a parameter just for testing.

The config value can be changed from the test file before the request is made https://github.com/Countly/countly-enterprise-plugins/blob/next/plugins/crashes-jira/tests.js#L62